### ISSUE TITLE
Question about documentation in example snippet

### ISSUE THREAD

**Issue by jamesblacklock** [mike-marcacci/node-redlock#253]
https://github.com/mike-marcacci/node-redlock/blob/b720d5fdbe5bb54316031aec8b2c15128dbfa8a9/src/index.ts#L637

Is bullet point (1) actually true? As far as I can tell from the code, the only way the signal gets aborted is if lock extension fails, and the `queue()` routine is called asynchronously with the `routine()`, so the duration of the `routine()` should not affect whether `extend()` is called.

**Comment by mike-marcacci** [comment]
You're correct, and that comment should be changed: the AbortController supplied by `using` will only be aborted if extension errors.
